jujugui/static/gui/src/app/components/service-overview/service-overview.js

```diff
@@ -39,35 +39,77 @@ YUI.add('service-overview', function() {
               value={action.value}
               valueType={action.valueType}
               link={action.link}
-              linkTitle={action.linkTitle}
-               />);
+              linkTitle={action.linkTitle} />);
       });
       return items;
     },
 
-    render: function() {
-      var actions = [
-        {
-          title: 'Units',
-          value: 5,
-          icon: '<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="16px" height="16px" viewBox="0 0 16 16"><path fillrule="evenodd" class="inspector-list__list-item-image" d="M 14.83 16C 14.83 16 10.17 16 10.17 16 9.53 16 9 15.47 9 14.83 9 14.83 9 10.17 9 10.17 9 9.53 9.53 9 10.17 9 10.17 9 14.83 9 14.83 9 15.47 9 16 9.53 16 10.17 16 10.17 16 14.83 16 14.83 16 15.47 15.47 16 14.83 16ZM 15 10.17C 15 10.07 14.93 10 14.83 10 14.83 10 10.17 10 10.17 10 10.07 10 10 10.07 10 10.17 10 10.17 10 14.83 10 14.83 10 14.93 10.07 15 10.17 15 10.17 15 14.83 15 14.83 15 14.93 15 15 14.93 15 14.83 15 14.83 15 10.17 15 10.17ZM 14.83 7C 14.83 7 10.17 7 10.17 7 9.53 7 9 6.47 9 5.83 9 5.83 9 1.17 9 1.17 9 0.53 9.53 0 10.17 0 10.17 0 14.83 0 14.83 0 15.47 0 16 0.53 16 1.17 16 1.17 16 5.83 16 5.83 16 6.47 15.47 7 14.83 7ZM 15 1.17C 15 1.07 14.93 1 14.83 1 14.83 1 10.17 1 10.17 1 10.07 1 10 1.07 10 1.17 10 1.17 10 5.83 10 5.83 10 5.93 10.07 6 10.17 6 10.17 6 14.83 6 14.83 6 14.93 6 15 5.93 15 5.83 15 5.83 15 1.17 15 1.17ZM 5.83 16C 5.83 16 1.17 16 1.17 16 0.53 16 0 15.47 0 14.83 0 14.83 0 10.17 0 10.17 0 9.53 0.53 9 1.17 9 1.17 9 5.83 9 5.83 9 6.47 9 7 9.53 7 10.17 7 10.17 7 14.83 7 14.83 7 15.47 6.47 16 5.83 16ZM 6 10.17C 6 10.07 5.93 10 5.83 10 5.83 10 1.17 10 1.17 10 1.07 10 1 10.07 1 10.17 1 10.17 1 14.83 1 14.83 1 14.93 1.07 15 1.17 15 1.17 15 5.83 15 5.83 15 5.93 15 6 14.93 6 14.83 6 14.83 6 10.17 6 10.17Z" fill="rgb(147,161,161)"></path></svg>'
-        },
-        {
-          title: 'Errors',
-          value: 5,
-          valueType: 'error',
-        },
-        {
-          title: 'Pending',
-          value: 5,
-          valueType: 'pending',
-        },
-        {
-          title: 'Uncommitted',
-          value: 5,
-          valueType: 'uncommitted',
-        },
-        {
+      /**
+        Parses the supplied unit data to return the status number.
+
+        @method _parseStatusData
+        @param {Array} units An array of units.
+      */
+      _parseStatusData: function(units) {
```

Comment:
This method is reproducing a bunch of the work also done in the added-services-list-item `_parseStatusData` It feels like this processing should be moved into the unit model so that it can be shared. You don't have to do this in this branch but if not can you create a follow-up card.